### DISCUSSION TITLE
Are prefix / suffix & formating for field number supported?

### DISCUSSION THREAD

**Discussion by Rar9** [payloadcms/payload#1222] [Roadmap]
I like to use number fields and like to display prefix like € and suffix for kg so the user just needs to enter a number

Is this  possible incl. location formating?

Like 1.000,00

**Comment by jmikrut** [comment]
Right now it's not, but this would be a great thing that we should add!

We would happily accept a PR to do so as well. Could be really good to have. I'll mark this as `planned`.

**Comment by phillipmalboeuf** [comment]
Hi @jmikrut @Rar9, I'm thinking of starting a PR for this that's using javascript's [NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat). It could also make use of Payload's admin I18n. Sounds good?

**Comment by phillipmalboeuf** [comment]
Hey gang @jmikrut @Rar9 @jacobsfletch, I've opened a PR here: https://github.com/payloadcms/payload/pull/1924
It's my second attempt at a PR for it (https://github.com/payloadcms/payload/pull/1510), this one is cleaner and the first one was closed because it made unwarranted use of FieldDescription.
